assets/src/stories-editor/components/inserter/menu.js

```diff
@@ -291,7 +291,6 @@ export class InserterMenu extends Component {
 				<div
 					className="editor-inserter__results amp__block-editor-inserter__results block-editor-inserter__results"
 					ref={ this.inserterResults }
-					tabIndex="0"
```

Comment:
@barklund this one was unclear to me if it should remain. 

Error: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-tabindex.md

Does `<div>` require and tabIndex?

Reply:
`stories-editor` package is the old editor. You can safely ignore anything inside that package.

Reply:
@barklund sorry, my question should have been related to the new `edit-story`  editor. Do we want to allow `noninteractive-tabindex`? Seems it was used several times in the old editor.

Reply:
No, all tabindex'ed nodes should be interactive either through choice of element or application of proper role.

Actually most tabindex'ed nodes in the new editor will be based off the `focusable` component I'm creating in #3826, and as both the role and the element is dynamic, it can't be statically validated.